test/data/resources-v0.json

```diff
@@ -0,0 +1,200 @@
+        ],
+        "ttl": 64
+      }
+    },
+    "NAME": {
+      "type": "CNAME",
+      "resource": {
+        "canonical": "foo.",
+        "ttl": 64
+      }
+    },
+    "CANONICAL": {
+      "type": "CNAME",
+      "resource": {
+        "canonical": "foo.bar@1.2.3.4",
+        "ttl": 64
+      }
+    },
+    "DELEGATE": {
+      "type": "DNAME",
+      "resource": {
+        "delegate": "bar.foo@4.3.2.1",
+        "ttl": 64
+      }
+    },
+    "NS": {
+      "_NOTE": "Includes glue A records",
+      "type": "NS",
+      "resource": {
+        "ns": [
+          "ns1.cloudflare.com@1.2.3.4",
+          "ns2.cloudflare.com@1.2.4.4"
+        ],
+        "ttl": 64
+      }
+    },
+    "SERVICE": {
+      "_NOTE": "Includes 2 SRV records, the target is a name or an IP",
+      "type": "SRV",
+      "resource": {
+        "service": [{
+          "service": "a2.",
+          "protocol": "lseed.",
+          "target": "1.2.3.4",
+          "priority": 0,
+          "weight": 5,
+          "port": 7868
+        }, {
+          "service": "a2.",
+          "protocol": "lseed.",
+          "target": "btcseeder",
+          "priority": 0,
+          "weight": 2,
+          "port": 1337
+        }],
+        "ttl": 64
+      }
+    },
+    "URI": {
+      "type": "URI",
+      "resource": {
+        "uri": ["ftp://ftp1.example.com/public"],
+        "ttl": 64
+      }
+    },
+    "EMAIL": {
+      "type": "RP",
+      "resource": {
+        "email": ["foo@gmail.com"],
+        "ttl": 64
+      }
+    },
+    "TEXT": {
+      "type": "TXT",
+      "resource": {
+        "text": ["this is a txt record"],
+        "ttl": 64
+      }
+    },
+    "LOCATION": {
+      "type": "LOC",
+      "resource": {
+        "location": [{
+          "version": 0,
+          "size": 0,
+          "horizPre": 1,
+          "vertPre": 2,
+          "latitude": 128,
+          "longitude": 256,
+          "altitude": 16
+        }],
+        "ttl": 64
+      }
+    },
+    "MAGNET": {
+      "type": "URI",
+      "resource": {
+        "magnet": [
+          "magnet:?xt=urn:btih:8c271f4d2e92a3449e2d1bde633cd49f64af888f"
+        ],
+        "ttl": 64
+      }
+    },
+    "DS": {
+      "type": "DS",
+      "resource": {
+        "ds": [{
+          "keyTag": 35215,
+          "algorithm": 13,
+          "digestType": 2,
+          "digest": "7C50EA94A63AEECB65B510D1EAC1846C973A89D4AB292287D5A4D715136B57A3"
+        }],
+        "ttl": 64
+      }
+    },
+    "TLS": {
+      "type": "TLSA",
+      "resource": {
+        "tls": [{
+          "protocol": "tcp.",
+          "port": 443,
+          "usage": 0,
+          "selector": 0,
+          "matchingType": 0,
+          "certificate": "0102348790"
+        }],
+        "ttl": 64
+      }
+    },
+    "SMIME": {
+      "type": "SMIMEA",
+      "resource": {
+        "smime": [{
+"hash": "92faa0601d44a3c2b98318d8f7e4cd3b17f25caa6753cc04365d753d",
```

Comment:
nit: indentation?

Reply:
Good catch, must have missed it in my editor when it overflowed to the next line